src/css/theme/_base.scss

```diff
@@ -15,6 +15,27 @@
 @import 'stats';
 @import 'toggle';
 
+.ais-clear-all,
+.ais-current-refined-values,
+.ais-hierarchical-menu,
+.ais-menu,
+.ais-pagination,
+.ais-price-ranges,
+.ais-range-slider,
+.ais-refinement-list,
+.ais-search-box,
+.ais-star-rating,
+.ais-stats,
+.ais-toggle {
```

Comment:
I did not know about this CSS3 pattern matching selector. There's a lot of divs created with `.ais-` prefix, do you know what is the best in term of performances?